connect/src/main/scala/org/apache/spark/sql/connect/service/SparkConnectService.scala

```diff
@@ -185,11 +186,10 @@ object SparkConnectService {
   /**
    * Starts the GRPC Serivce.
    *
-   * TODO(SPARK-40536) Make port number configurable.
    */
   def startGRPCService(): Unit = {
     val debugMode = SparkEnv.get.conf.getBoolean("spark.connect.grpc.debug.enabled", true)
-    val port = 15002
+    val port = SparkEnv.get.conf.getInt(CONNECT_GRPC_BINDING_PORT.key, 15002)
```

Comment:
This means we are duplicating the default value `15002` in 2 places. `SparkConf.get` is a better API to use
```
  def get[T](entry: ConfigEntry[T]): T = {
    recordConfigEntryAccessDuringWarmUp[T](entry, false, this) {
      entry.readFrom(reader)
    }
  }
```

Reply:
Yes this is what I confused on the `getInt` API which always ask for a default value, and the get, instead, return a string.

The difference of this get is based on the Entry parameter, which was ignored when I searched the API.

Done.